### PR TITLE
Customizing the logger

### PR DESCRIPTION
**Pull request by danthorpe**

### CODE REVIEW

File: examples/Permissions/Permissions/AppDelegate.swift [hunk 1]
```diff
@@ -16,8 +19,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+            case .Info:
+                log.info(message, file, function, line: line)
+            case .Warning:
+                log.warning(message, file, function, line: line)
+            case .Fatal:
+                log.error(message, file, function, line: line)
+            }
+        }
+        
+        LogManager.severity = .Info
```
Comment (danthorpe) [inline examples/Permissions/Permissions/AppDelegate.swift]: Okay, so here is how to configure the internal logger to use an external logging framework such as SwiftyBeaver.

And so now logging output looks like this....

custom logging

The only thing that is perhaps a little bit annoying, is that the various severity levels might not always match up. Operations defines:

```swift
public enum LogSeverity: Int, Comparable {

    /// Chatty
    case Verbose = 0

    /// Public Service Announcements
    case Notice

    /// Info Bulletin
    case Info

    /// Careful, Errors Occurring
    case Warning

    /// Everything Is On Fire
    case Fatal
}
```

And, SwiftyBeaver defines:

```swift
    public enum Level: Int {
        case Verbose = 0
        case Debug = 1
        case Info = 2
        case Warning = 3
        case Error = 4
    }
```

So, you'll have to match things up...

**Comment by danthorpe** [comment]
Hi @shsteven - what are your thoughts on this? See the changes in the AppDelegate...

**Comment by shsteven** [comment]
Looks great!

Matching up the logging levels is fine, as long as all logs are captured.

Do you mind adding a short description to README? Something like

> Integrating with other logging libraries
> LogManager can be set up to pipe logs to other logging libraries, such as XCGLogger or SwiftyBeaver. Sample code is available in AppDelegate.swift in the Permissions example app.

**Comment by shsteven** [comment]
#144

**Comment by danthorpe** [comment]
@shsteven agreed that it needs to be documented - I'm working on overhauling the documentation separately and will have a place for all of this.

**Pull request merged**